app/controllers/admin/events_controller.rb

```diff
@@ -92,7 +95,27 @@ def update
       end
     end
 
-    def create; end
+    def create
+      @url = admin_conference_program_events_path(@conference.short_title, @event)
+      @users = User.all.order(:name)
+      @languages = @program.languages_list
+      @event.submitter = current_user
+      @event.state = :confirmed
+
+      if @event.save
+        ahoy.track 'Event submission', title: 'New submission'
+        redirect_to admin_conference_program_events_path(@conference.short_title), notice: 'Event was successfully submitted.'
+      else
+        flash[:error] = "Could not submit proposal: #{@event.errors.full_messages.join(', ')}"
+        render action: 'new'
```

Comment:
I am aware of this issues, but don't have simple and elegant solution for this ATM.
@lagartoflojo maybe you have some thoughts on the elegant the solution for this?
The thing is that form loses its nested attributes on re-render, since the event.speakers is not yet persisted, the preselected values are lost for the speakers dropdown on the form.